libraries/tickv/src/async_ops.rs

```diff
@@ -147,7 +149,7 @@ pub struct AsyncTicKV<'a, C: FlashController<S>, const S: usize> {
     pub tickv: TicKV<'a, C, S>,
     key: Cell<Option<u64>>,
     value: Cell<Option<&'static mut [u8]>>,
-    buf: Cell<Option<&'static mut [u8]>>,
```

Comment:
This fails to build, there are still users of this

Reply:
Yeah I'm trying to balance reviewable PRs with reasonable programming overhead.

#3508 if you want to see the full changes that pass CI.